src/main/java/com/squareup/kotlinpoet/CodeWriter.kt

```diff
@@ -195,13 +195,13 @@ internal class CodeWriter @JvmOverloads constructor(
   }
 
   @Throws(IOException::class)
-  fun emit(s: String): CodeWriter = emitAndIndent(s)
+  fun emitCode(s: String) = emitCode(CodeBlock.of(s))
```

Comment:
I renamed this because we had two methods, `emit(String)` and `emit(String, vararg)` that had different behavior and that was really weird for the calling code. The vararg one interpreted its input as code; the non-vararg didn’t.